config/linux-librem_common-6.1.8.config

```diff
@@ -1252,10 +1252,12 @@ CONFIG_BLK_DEV_DM=y
 # CONFIG_DM_DEBUG is not set
 CONFIG_DM_BUFIO=y
 # CONFIG_DM_DEBUG_BLOCK_MANAGER_LOCKING is not set
+CONFIG_DM_BIO_PRISON=y
+CONFIG_DM_PERSISTENT_DATA=y
 # CONFIG_DM_UNSTRIPED is not set
 CONFIG_DM_CRYPT=y
 # CONFIG_DM_SNAPSHOT is not set
```

Comment:
This should be enabled and also forced upon all other linux configs that would miss this and other options enabled here if missing please. If legacy breaks, that will be the end of them.

Reply:
It looks like only librem_common-6.1.8 and nitropad-x are missing it.  We discussed this separately, I'm going to re-make the 6.1.8 config from the librem_common 5.10 config since a lot of cleanup was done there that wasn't carried over to the 6.1.8 config.  Will check that out, I'll pile on here if the change is reasonable or open a separate PR if it's a major change.

Reply:
@daringer @jans23 Please note that nitropad shared linux config will need changes accordingly to be able to not bug if one uses function which will be available to all.

Reply:
I think we need a way to describe "config Heads needs" from board-specific config like drivers.  This is pretty hard to maintain right now, and it's not really clear what config Heads needs, best I can think to do is diff some boards' kernel configs and go through the result.

I'm not really sure what we could do offhand though :thinking:

Reply:
thanks for pointing that out,  created a [reminder](https://github.com/Nitrokey/heads/issues/36) for us...

Reply:
@tlaurion #1590 rebuilds the librem-common-6.1.8 config; rebased this PR on that.